staging/src/k8s.io/component-helpers/resource/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-node-approvers
```

Comment:
IIUC from the discussion in the other thread with @SergeyKanzhelev and @liggitt,
approvers should be `api-approvers` only.

Reviewers can be `sig-node-reviewers`, `sig-scheduling` (this is the reviewers list... yes it should be renamed).

Labels: `sig/node` and `sig/scheduling`

Reply:
Updated